cms/settings/base.py

```diff
@@ -170,12 +173,41 @@
 
 
 # Database
-# This setting will use DATABASE_URL environment variable.
-# https://docs.djangoproject.com/en/stable/ref/settings/#databases
-# https://github.com/kennethreitz/dj-database-url
 
-DATABASES = {"default": dj_database_url.config(conn_max_age=600, default="postgres:///ons")}
+if "PG_DB_ADDR" in env:
+    # Use IAM authentication to connect to the Database
+    DATABASES = {
+        "default": cast(
+            dj_database_url.DBConfig,
+            {
+                "ENGINE": "django_iam_dbauth.aws.postgresql",
+                "NAME": env["PG_DB_DATABASE"],
+                "USER": env["PG_DB_USER"],
+                "HOST": env["PG_DB_ADDR"],
+                "PORT": env["PG_DB_PORT"],
+                "CONN_MAX_AGE": 870,  # Just under 15 minutes, to match password expiry
```

Comment:
Nit: Let's make this a config with a default.

Reply:
https://github.com/ONSdigital/dis-wagtail/pull/15/commits/3c4ca21b9d3d3b2f288b219f8686cce81aacd78a